Header header
int16 remainingHP
int16 attackArmorID
int16 bulletCount
float32 gimbalAngleYaw
float32 gimbalAnglePitch
float32 bulletSpeed
int16 RFID
